include/dpp/emoji.h

```diff
@@ -194,6 +194,16 @@ class DPP_EXPORT emoji : public managed, public json_interface<emoji> {
 	 */
 	emoji& load_image(std::string_view image_blob, const image_type type);
 
+	/**
+	 * @brief Load an image into the object
+	 *
+	 * @param image_blob Image binary data
+	 * @param type Type of image. It can be one of `i_gif`, `i_jpg` or `i_png`.
+	 * @return emoji& Reference to self
+	 * @throw dpp::length_exception Image content exceeds discord maximum of 256 kilobytes
+	 */
+	emoji& load_image(const std::byte* data, uint32_t size, const image_type type);
```

Comment:
why are we using std::byte instead of e.g. uint8_t?
std::byte has a very limited number of operators associated with its wrapper class which make it very difficult to do basic things with at times.

Reply:
Also dpp::utility::read_file does not return an array of std::byte, it returns string.